util/pdapi/const.go

```diff
@@ -16,12 +16,13 @@ package pdapi
 
 // The following constants are the APIs of PD server.
 const (
-	HotRead    = "/pd/api/v1/hotspot/regions/read"
-	HotWrite   = "/pd/api/v1/hotspot/regions/write"
-	HotHistory = "/pd/api/v1/hotspot/regions/history"
-	Regions    = "/pd/api/v1/regions"
-	RegionByID = "/pd/api/v1/region/id/"
-	Stores     = "/pd/api/v1/stores"
-	Status     = "/pd/api/v1/status"
-	Config     = "/pd/api/v1/config"
+	HotRead      = "/pd/api/v1/hotspot/regions/read"
+	HotWrite     = "/pd/api/v1/hotspot/regions/write"
+	HotHistory   = "/pd/api/v1/hotspot/regions/history"
+	Regions      = "/pd/api/v1/regions"
+	RegionByID   = "/pd/api/v1/region/id/"
+	StoreRegions = "/pd/api/v1/regions/store/"
```

Comment:
Prefer to keep the same format, e.g. `/pd/api/v1/regions/store`

Reply:
Done, and hence add ` + "/"` in implemention code 
```go
pdapi.StoreRegions+"/"+strconv.FormatUint(storeID, 10), nil, &regionsInfo)
pdapi.RegionByID+"/"+strconv.FormatUint(regionID, 10), nil, &regionInfo)
```